packages/Tokens/src/tokens.yaml

```diff
@@ -148,6 +148,10 @@ popover:
   border-radius: $card--border-radius
   shadow: 0 2px 4px rgba(0, 0, 0, 0.1)
 #
+pill:
+  small-radius: 8px / 50%
+  medium-radius: 12px / 50%
```

Comment:
I think this would be clearer as `4px` and `6px` 😆

Reply:
We don't want that as, border radius is a bit special. It doesn't look right with one value, with this above syntax we get elliptical radius which is what we want

Reply:
TIL that the CSS `border-radius` property is more advanced than I thought!

Reply:
Was it not a joke? 😄 
Also 8px / 50% is 16px 🤓 :trollface: